src/block-management/block-toolbar.js

```diff
@@ -49,8 +49,9 @@ export class BlockToolbar extends Component<PropsType> {
 					<Toolbar>
 						<ToolbarButton
 							label={ __( 'Add block' ) }
-							icon="insert"
+							icon="plus-alt"
 							onClick={ onInsertClick }
+							extraProps={ styles.addBlockButton }
```

Comment:
We shouldn't be setting the icon's `color` attribute as it will be prone to error if we add css rules to this `addBlockButton` class. Also it's best practice to group styling in style or className.
So:
```
extraProps={ { style: styles.addBlockButton } }
```